tests/queries/0_stateless/03704_rmv_logging_internal_queries.sh

```diff
@@ -31,8 +31,8 @@ $CLICKHOUSE_CLIENT --query "SYSTEM WAIT VIEW rmv_test" 2> /dev/null
 $CLICKHOUSE_CLIENT --query "SYSTEM FLUSH LOGS query_log"
 $CLICKHOUSE_CLIENT --query "
 SELECT
-    countIf(query LIKE '%INSERT INTO $CLICKHOUSE_DATABASE.\`.tmp.inner_id.%' AND type = 'QueryStart'),
-    countIf(query LIKE '%INSERT INTO $CLICKHOUSE_DATABASE.\`.tmp.inner_id.%' AND type = 'QueryFinish'),
+    countIf(query LIKE '%INSERT INTO $CLICKHOUSE_DATABASE.\`.tmp.inner_id.%' AND type = 'QueryStart') >= 1,
+    countIf(query LIKE '%INSERT INTO $CLICKHOUSE_DATABASE.\`.tmp.inner_id.%' AND type = 'QueryFinish') >= 1,
```

Comment:
`$CLICKHOUSE_DATABASE` should make the insert queries unique, so I don't think this is the good approach.